util/dockerfiles/github-ci/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# This is a container used to run CI jobs on GH
+FROM ubuntu:22.04
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update -y && apt-get install -y --no-install-recommends \
+    bash \
+    clang-13 \
+    cmake \
+    cmake \
```

Comment:
Can you add `cscope` too so we don't need to download that for `check_annotations_rt_calls`

Reply:
Yep! Missed that, but luckily left myself a duplicate `cmake` which is in the perfect spot to change to `cscope`!